### ISSUE TITLE
[Stripe] add StripeJS functionality to admin 'new payment' section

### ISSUE THREAD

**Issue by oeoeaio** [openfoodfoundation/openfoodnetwork#1745]
### Overview
Payments can be manually added to orders by enterprises users from the admin section. This does not apply to all payment methods (PayPal Express for instance), however it would be ideal to allow the creation of payments on behalf of customers using Stripe. To do this, we need to use StripeJS to submit card information to Stripe and then use the same token-based flow to create and process a new payment as we use in the front-end.

This issue was spawned out of the investigation that took place as part of #1728.

### Tasks
- [ ] Add StripeJS library to the 'new' view, use it to request a token and then submit it with the form
- [ ] Probably going to require some kind of override of the update button in that view...
- [ ] Make sure that the admin PaymentsController is able to handle the token properly

**Comment by daniellemoorhead** [comment]
Given StripeJS is going to be replaced by Elements ( #1755 ) is this issue impacted in terms of the defined solution? Ping @oeoeaio @stveep

**Comment by stveep** [comment]
I can implement the Elements form on this page instead, the rest of the issue will be as above

**Comment by daniellemoorhead** [comment]
@stveep I spoke to @oeoeaio today and this isn't on the critical path for releasing Stripe first release. If you get a chance to do it then yay, we'll either add it into this release (which will hopefully be this week) or make it part of the dot point release for the end of september.

tl;dr: no rush on this, at your leisure :)

**Comment by daniellemoorhead** [comment]
Merged #2574 so closing this.
